### ISSUE TITLE
Boon Focus Tier 2 Skips move action

### ISSUE THREAD

**Issue by jjnich** [openlegend/core-rules#55]
"Additionally, invoking the boon is one time increment faster. If the invocation time is a major action, it becomes a minor action..."
Shouldn't a major action become a move action, a move action become a minor action, so-on.

**Comment by DMsShadow** [comment]
I wouldn't say so. It's meant to make the action quicker, and a move action isn't quicker than a major action, they're just different kinds of "single" actions in a way.

**Comment by brianfeister** [comment]
That's an interesting point, and it's a bit complex. Basically, for 6 points, you should be getting ALOT of mileage out of the feat. I suppose one could consider making Boon Focus cost 2 points instead, so that the ability to invoke as a minor action still costs 6, but you could optionally spend only 4 points. The problem is that if we move that direction, we will need to give the feat 4 tiers instead of 3. 

Tiers 2 & 3 would each speed invocation up by one increment, but the problem is what becomes of the `Teleport` boon which is already a move action. Tier 3 (the second invocation time reduction) wouldn't do anything for that boon.

When I started reading this, I was inclined to agree with you, but the more I write, the more I feel that it actually works OK as it is now. I've spent alot of time GMing with some players utilizing this feat and I think it maybe works fine as is -- give it a try in actual play and let me know if you have further thoughts.

**Comment by jjnich** [comment]
This came up when I was building out a teleporting monk (Read: Nightcrawler) if I get boon focus 2 it makes my teleport, which is normally a move action, go down to a minor action, correct? Then boon focus 3 makes it "always active"? not sure what exactly that means for a "duration: instantaneous" boon.

Totally unrelated question - - - Can I target an ally with Teleport? Can I target a group with Teleport?

**Comment by brianfeister** [comment]
@jjnich - Yeah, duration instantaneous boons with Boon Focus 3 are an interesting question. I'm open to suggestions. @istabosz any thoughts on this? I might argue that it's self-evident that instantaneous boons can't benefit from Boon Focus 3. Perhaps we simply comment on it and let people know that "In case you're wondering, yes Boon Focus 3 does nothing for instantaneous boons".

About teleporting allies and groups - for allies, yes the boons all adhere to the same rules, which is that you can target yourself or an ally with them. Single-target invocation is an auto-success. Group invocation gets advantage if you have Boon Focus, as noted in [the feat description](http://www.openlegendrpg.com/feats/boon_focus)

**Comment by jjnich** [comment]
Assuming boon focus 2 works for lowering a move action to minor action (which is questioned only because move action isn't included in the description for the feat), Maybe boon focus 3 should have something allowing you to do a boon multiple times as a minor action? or twice as a move action?

For example if I drop 9 feat points into boon focus (teleport) I think I should be able to teleport AT LEAST twice in one turn, but RAW it being a minor action I can't because you can't do the same minor action more than once in a turn.

Other example, boon focus (aid) would be really cool to be able to help 2 (or even 3) allies at once because you're just SUPER GOOD AT IT!

Third example, boon focus (heal) you become a super healer and can now use this boon twice and either heal one person double or heal 2 team members within reach.

I think it would be acceptable as a minor action to use it once and +X action level (minor->move->major->minute->hour->day) to use the boon X times. heal once as a minor, or heal 2 allies and only use a move action, or heal 3 allies and use your major action rushing between them with your move.

Just a thought

**Comment by raihje** [comment]
With the above if the teleport is a minor action already, you could teleport somewhere between 5 and 25 feet and then move an additional 30 feet on foot. Teleporting twice would be nice for flavour, letting you use the move action and minor action for Teleport. However having that apply to all instantanous boons could be problematic / a bit too powerful? Healing twice, or Aiding Twice, Stacking Haste, etc.. This could be difficult to balance with other feats if they did that.

**Comment by brianfeister** [comment]
Actually @jjnich, you're missing something important there - Teleport is a move action, so reducing it to a minor means that you can teleport as a move action and ALSO as a minor action. I see where the miscommunication comes from, but I think this might just be a question of semantics and I think it may already be well-worded. You can spend a Major action to take an extra move action. In the same way, you can Teleport as a minor action, but also choose to use a move action, thus teleporting twice in one round. And I'm pretty sure I don't want people to teleport 3x per round.

@jjnich - helping target many people is already a feat, it's called [Multi-Target Boon Specialist](http://www.openlegendrpg.com/feats/multi-target_boon_specialist)

**Comment by jjnich** [comment]
Oh yeah I forgot about Multi-Target Boon Specialist! Just trying to come up with something that would make Tier 3 worth it for an instantaneous boon. Maybe just including a note (Instantaneous boons don't persist) is the best course of action.

Side-Note: Teleporting 3x in a round just makes my Nightcrawler build way cooler xD

**Comment by jjnich** [comment]
What I would do to clear it up is to change "Additionally" in the tier 3 section to "For boons with a duration longer than 'Instantaneous'..."

**Comment by brianfeister** [comment]
@jjnich XD !!! You can teleport 3x if you want: Major + Move + Minor ;D

**Comment by brianfeister** [comment]
Sounds like a good clarification, thanks :)

**Comment by jjnich** [comment]
Seems like this is done. Closing issue.

**Comment by brianfeister** [comment]
Cool - going to re-open as a reminder because we still need to update the rules text ;)

**Comment by istabosz** [comment]
Just updated the clarification in Tier 3. Kinda hard to follow this conversation, was there something that needed to be done for tier 2?

**Comment by brianfeister** [comment]
Nah, just clarify at Tier 3. But the "neat and tidy" person in me does wish we could present an alternate benefit for Tier 3 instantaneous boons.... can't come up with anything though ....

**Comment by istabosz** [comment]
It's tough to come up with another benefit for individual boons, let alone one that is generalizable to more than one boon. The only thing I can think of is making the advantage even higher for boons with a duration other than "sustain persists". But that seems like overkill.

I guess one other possibility is to increase the attribute die by one. That's a benefit that is generally hard to come by.

**Comment by brianfeister** [comment]
Maybe if it's instantaneous, the equivalent of being "non-dismissable" is that you can invoke it and even cut through anti-magic or counter-magic. So, Boon Focus 3 (Teloport) makes you so powerful that your teleport can't be countered or prevented.

**Comment by istabosz** [comment]
Would that apply to all boons or just instantaneous ones?

**Comment by jjnich** [comment]
Yeah it'd be better to come up with something that obviously only hits instantaneous boons. I'll keep thinking about this and try to come up with a couple better options.

**Comment by istabosz** [comment]
Thinking more about it, I kinda like increasing the advantage for instantaneous boons. I bet the most common instantaneous boon that will be chosen for this tier is Heal, so more advantage means either more healing or more targets. So, my proposition is:

If the boon has a duration of "sustain persists", you gain advantage 4 when multi-targeting and the boon is always active on yourself.

If the boon has a duration other than "sustain persists", you gain advantage 5 when multi-targeting.

**Comment by jjnich** [comment]
I like that!

**Comment by brianfeister** [comment]
Sure, I'm fine with that :)

**Comment by DukeAnax** [comment]
It totally makes sense. The one advantage you don't have for sustain boons is the multitargetting yourself.

There is one other issue though, there are several Boons, like Shapeshifter and Animation that don't make sense to be "always active for you"

Maybe instead of writing "the boon is always active for you" you could write "the boon can always be activated for you."

There is one other issue though, there is at least one Boon, Shapeshifter that doesn't make sense to be "always active for you" maybe just write "the boon doesn't require a minor action to sustain on yourself.

Maybe a bit off topic, but: if I shapeshift someone can they sustain the boon themselves, or do I have to do that? If it's the later, maybe you could add to the Tier III that you can sustain the Boon on several targets with a minor action.

**Comment by istabosz** [comment]
@brianfeister I made the change. Can you check the formatting, as my YAML ain't too strong.

@DukeAnax The caster is always the one who sustains the boon. They can sustain it on however many targets they included in the original casting with a single minor action.

**Comment by DukeAnax** [comment]
@istabosz Ah Ok, thank you. maybe you should clarify that in the Boon Sustain section.

**Comment by raihje** [comment]
WHen you say that you can invoke the boon at all three actions (Major, Minor, Move) how so? The Boon itself is a Move action (standard use) then you get a Minor Action use from the Boon Focus but in order to up-scale the time it takes, wouldn't you need Ritual Casting?

**Comment by jjnich** [comment]
You can use a move action in place of your major action allowing you to invoke the boon the second time as a move action. There is no ritual casting in this system as far as I know.

**Comment by brianfeister** [comment]
@raihje The feat we're discussing `Boon Focus` reduces the invocation time increment by one step. So, Boon Focus 2 (Teleport) lets you teleport as a minor action. You can "trade down" on your actions. So, given that teleport is a minor action, you can spend a minor to do it once, then choose to trade your Move action down to a minor for a second invocation, and then use your major action, trading down again to a minor action.

**Comment by raihje** [comment]
Ah, as I understood it, it was A minor action to Teleport via the feat, a Move action to teleport via the regular action. Then the Major Action taking the "Move Action" to use Teleport as it's normal action (this would be 3 teleports, but no movement or other actions other than sustaining a boon/other minor stuff) So basically the same idea, you aren't forced to use Teleport as a minor action via the feat, it's just an option to. Thanks.

**Comment by brianfeister** [comment]
Cool :) ... I think we have it explicitly on the site already. I mean, we're not going to point out that if the feat hastens your invocation by an increment that you can still do it at the usual invocation time. That's one of those things where we hope common sense will prevail at most gaming tables, but we aren't going to caveat this sort of thing in every place it appears.

```
On your turn, you may take…
1 Major Action*
1 Move Action
Any number of minor actions

* Major action may be traded for a move action
```

**Comment by brianfeister** [comment]
@jjnich Open Legend doesn't have default "ritual casting" but we do have a `Ritual Magic` feat that allows you to heighten a spell or accomplish things that can't be done with the temporary nature of bane and boons invocation: http://www.openlegendrpg.com/feats/ritual_magic

**Comment by istabosz** [comment]
I think everything here has been accomplished.
